gateway-service/src/main/resources/application.yml

```diff
@@ -51,6 +51,10 @@ apiml:
             jwtKeyAlias: jwtsecret
         x509:
             enabled: true
+            useZss: false
+            externalMapperUrl: http://localhost:8542/certificate/x509/map
+
+
```

Comment:
Could we simplify this?

Reply:
I can't see the change in internal repo, please help me, i must be missing something.